### PR TITLE
Price Database: Inserting help buttons and linking them to the section in the manual

### PR DESCRIPTION
**Pull request by CWehli**

### CODE REVIEW

File: gnucash/gnome/dialog-price-edit-db.c [hunk 1]
```diff
@@ -782,6 +792,8 @@ gnc_prices_dialog_create (GtkWidget * parent, PricesDialog *pdb_dialog)
         gtk_widget_grab_default (button);
         gtk_widget_grab_focus (button);
 
+        button = GTK_WIDGET(gtk_builder_get_object (builder, "help_button"));
```
Comment (fellen) [inline gnucash/gnome/dialog-price-edit-db.c:795]: My feeling says, it should be earlier in the list.
@Bob-IT Is there a recommend order?

**Review by fellen — COMMENTED** [review]
I would prefer a unification of the link IDs.
Forearding the rest of the decision to @Bob-IT

File: gnucash/gnome-utils/gnc-ui.h [hunk 2]
```diff
@@ -64,6 +64,9 @@
 #define HL_COMMODITY         "tool-commodity"
 #define HL_FIND_TRANSACTIONS "tool-find"
 #define HL_FIN_CALC          "tool-calc"
+#define HL_PRICE_DB          "tool-price"
+#define HL_PRICE_EDIT        "invest-stockprice-manual"
+#define HL_PRICE_DEL         "invest-stockprice-delete"
```
[the remark this answers is not in the document]
Comment (Bob-IT) [reply gnucash/gnome-utils/gnc-ui.h:69]: I am assuming `HL_PRICE_DEL` is another help button you are working on as it does not exist in code.

File: gnucash/gnome/dialog-price-edit-db.c (hunk 1 above)
Comment (Bob-IT) [reply gnucash/gnome/dialog-price-edit-db.c:795]: This line is not required.

**Comment by Bob-IT** [comment]
@fellen 
I am not sure about one help button placement..
Screenshot_2022-07-24_10-09-17
Should the close and help be swapped or maybe this is a better layout...
Screenshot_2022-07-24_10-32-54
.

File: gnucash/gnome-utils/gnc-ui.h (hunk 2 above)
[the remark this answers is not in the document]
Comment (CWehli) [reply gnucash/gnome-utils/gnc-ui.h:69]: Yes, you are right. I have this chapter already in german and do the backport the next time.

**Comment by CWehli** [comment]
> @fellen I am not sure about one help button placement.. [...] Should the close and help be swapped or maybe this is a better layout...  .

I prefer the second picture. How can i achieve this?

File: gnucash/gnome-utils/gnc-ui.h (hunk 2 above)
[the remark this answers is not in the document]
Comment (CWehli) [reply gnucash/gnome-utils/gnc-ui.h:69]: These two entries are supposed to point to `<sect2 invest-stockprice-...>` sections in the manual. Unfortunately, these targets are not jumped to correctly.

[the remark this answers is not in the document]
Comment (fellen) [reply gnucash/gnome-utils/gnc-ui.h:69]: From https://wiki.gnucash.org/wiki/Documentation_Improvement#Conventions:
**ID attributes**
    Many elements can contain an inside of the document unique id="**chapter**-more-specific-context" attribute.

Here the chapter is tools, not invest.

[the remark this answers is not in the document]
Comment (CWehli) [reply gnucash/gnome-utils/gnc-ui.h:69]: I correct the targets in the manuals with [PR #281](https://github.com/Gnucash/gnucash-docs/pull/281)

[the remark this answers is not in the document]
Comment (fellen) [reply gnucash/gnome-utils/gnc-ui.h:69]: … which is merged, so you can update the links here.

**Comment by fellen** [comment]
> @fellen
> I am not sure about one help button placement..

The 2. loos nicer.

**Comment by Bob-IT** [comment]
> I prefer the second picture. How can i achieve this?

@CWehli
Here is a patch for the glade file on [gist](https://gist.github.com/Bob-IT/ce92e24bdce1e07ec8abca3704288cd6)
I also think you should remove the `HL_PRICE_DEL` and add it to a new PR that adds the help button also.
If you rebase and push I will check it when available.

**Comment by CWehli** [comment]
@Bob-IT 
> Here is a patch for the glade file 

I've applied the patch, thanks for it. Please have a look.

**Comment by Bob-IT** [comment]
Thanks.

**Pull request merged**